Mathlib/Analysis/BoxIntegral/Partition/Additive.lean

```diff
@@ -48,7 +48,13 @@ structure BoxAdditiveMap (ι M : Type*) [AddCommMonoid M] (I : WithTop (Box ι))
     ∑ Ji in π.boxes, toFun Ji = toFun J
 #align box_integral.box_additive_map BoxIntegral.BoxAdditiveMap
 
+
+/-- A function on `Box ι` is called box additive if for every box `J` and a partition `π` of `J`
+we have `f J = ∑ Ji in π.boxes, f Ji`. -/
 scoped notation:25 ι " →ᵇᵃ " M => BoxIntegral.BoxAdditiveMap ι M ⊤
+
+/-- A function on `Box ι` is called box additive on subboxes of `I : Box ι` if for every subbox `J`
+of `I` and a partition `π` of `J` we have `f J = ∑ Ji in π.boxes, f Ji`. -/
```

Comment:
Why not inherit the docstring here?

Reply:
The original docstring explains two use cases which are then each wrapped in a different notation, although currently one of the notations is the general case which I'm not sure is what @urkud meant.

Reply:
I wanted this notation to be restricted to `I : Box ι`, not `I : WithTop (Box ι)`. But changing this is probably out of the scope of this PR (though if you'll try to add `WithTop.some` and see if something breaks, it would be nice).

Reply:
Okay that's what I thought. So you agree my docstrings are correct (if not quite matching reality right now)?

Reply:
Actually, you do use it as a general notation below, so I'll inherit the doc for this one.